Docs: Add config settings for fonts in reporting (#30421)
grafana/grafana: docs/sources/enterprise/reporting.md

> To modify the panels' clarity you can set a scale factor for the rendered images. A higher scale factor is more legible but will increase the file size of the generated PDF.

Reviewer: ```suggestion
To make a panel more legible, you can set a scale factor for the rendered images. However, a higher scale factor increases the file size of the generated PDF.
```

> It's also possible to specify custom fonts supporting different Unicode scripts. DejaVu that is the default font used for PFD rendering.

Reviewer: ```suggestion
You can also specify a custom font that support different Unicode scripts. The DejaVu font is the default used for PDF rendering.
```

Reviewer: Can a user specify "a" font or "any" font that fulfills that condition? I left it as "a" here because I assume that would otherwise require testing all all possible fonts.

Author: I wanted to stress that there's an option for specifying font different from the default one. The default font does not support, for example, Chinese glyphs. So having this option users are able to bypass the limitation of the default font by using some other one.

# Set the scale factor for rendering images. 2 is enough for monitor resolutions
# 4 would be better for printed material. Setting a higher value affects performance and memory
image_scale_factor = 2
# Path to the directory containing font files
fonts_path =

> # Name of the TrueType font file with Regular style

Reviewer: ```suggestion
# Name of the TrueType font file with regular style
```

What is the difference in meaning between Regular and regular? Similar comment for content in lines 140 and 142.

Author: There's no difference in meaning. It's just a font's style. You're right, will change to lowercase.
